plugins/panels/model_evaluation/__init__.py

```diff
@@ -504,6 +505,24 @@ def load_view(self, ctx):
         field = view_options.get("field", None)
         missing = ctx.panel.get_state("missing", "(none)")
 
+        # Restrict to subset, if applicable
+        subset_name = view_state.get("subset_name", None)
```

Comment:
This *should* wire up view callbacks for the ME panel, assuming that:
- A `subset_name` parameter is set that encodes the name of the subset (bar) that the user clicked on
- A `subsets` dict is available in panel state that encodes the user's current subsets definition, using the structure described in the `Subsets specification` section of the PR description

Reply:
cc @imanjra